modules/control/main.tf

```diff
@@ -261,6 +274,22 @@ resource "aws_security_group" "public_ssh" {
         cidr_blocks 	= ["0.0.0.0/0"]
     }
 
+
+    ingress {
+        from_port 	= 80
+        to_port 	= 80
+        protocol 	= "tcp"
+        cidr_blocks 	= ["10.0.0.0/16"]
+    }
+
+
+    ingress {
+        from_port 	= 443
+        to_port 	= 443
+        protocol 	= "tcp"
+        cidr_blocks 	= ["10.0.0.0/16"]
+    }
```

Comment:
This will open up 80 & 443 on all instances that use public_ssh. Instead we should create a second security group that has just these two ingress rules and use them below

Reply:
Also the cidr_blocks shouldn't be hard coded, better to use the `var. internal_cidr_blocks`.